index.html

```diff
@@ -199,12 +237,12 @@ <h4 class="footer__button__text">Get my resume</h4>
        
         <nav>
             <ul class="socials">
-                <li href=""><img src="./images/stackoverflow.svg" alt="go to my stackoverflow" class="social"></li>
-                <li href=""><img src="./images/github.svg" alt="go to my github" class="social"></li>
-                <li href=""><img src="./images/twitter.svg" alt="go to my twitter" class="social"></li>
-                <li href=""><img src="./images/linkendin.svg" alt="go to my linkendin" class="social"></li>
-                <li href=""><img src="./images/m.svg" alt="go to my M" class="social"></li>
-                <li href=""><img src="./images/fingers.svg" alt="go to my M" class="social"></li>
+                <li href="#"><img src="./images/stackoverflow.svg" alt="go to my stackoverflow" class="social"></li>
+                <li href="#"><img src="./images/github.svg" alt="go to my github" class="social"></li>
+                <li href="#"><img src="./images/twitter.svg" alt="go to my twitter" class="social"></li>
+                <li href="#"><img src="./images/linkendin.svg" alt="go to my linkendin" class="social"></li>
+                <li href="#"><img src="./images/m.svg" alt="go to my M" class="social"></li>
+                <li href="#"><img src="./images/fingers.svg" alt="go to my M" class="social"></li>
             </ul>
         </nav>
```

Comment:
- [ ] Kindly try to match the design in Figma. The width of the footer should be `100vw`, the text size and the icons size is not the same, the icon next to the text is out of the container while it should be next to the text. Also, there is no space below the footer 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/200125456-01c1eb15-4582-42be-a4da-e2300ca9ac0f.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/200125463-47d493e4-11d0-4c0f-b351-4ddc85b0f388.png)

Reply:
Fixed.